### ISSUE TITLE
Failed to install Telemetry !!!!!

### ISSUE THREAD

**Issue by MOHAMMADHMEAD** [nuxt/nuxt.js#10812]
Versions

- nuxt: 2.15.7
- node: 14 lts
- node_env: production

when trying to create a new nuxt app or try to install node modules  I got an error .

```
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod 
'/......./node_modules/@nuxt/telemetry/bin/nuxt-telemetry.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /......npm/_logs/2022-11-16T13_14_10_594Z-debug.log
```

@manniL

**Comment by zhiqiang-arctomsci** [comment]
Same

**Comment by AntoineAA** [comment]
same, nuxt 2.15.8

**Comment by baccouche-sami** [comment]
Same

**Comment by MOHAMMADHMEAD** [comment]
@pi0 any solution ?

**Comment by zhiqiang-arctomsci** [comment]
In https://github.com/nuxt/telemetry/issues/88

**Comment by pi0** [comment]
THere was an issue with npm while releasing `1.x` version for Nuxt 2. Should be fixed in `@nuxt/telemetry@1.3.9` please try upgrading lockfile.

**Comment by NigmaX** [comment]
Can confirm it was fixed! thanks
